services/vaults/holdings/save/handler.js

```diff
@@ -0,0 +1,111 @@
+/* 
+
+Holdings calculated here are for Vaults, Strategies and the Earn yTokens
+This is part of the TVL calculation defined here: https://hackmd.io/@dudesahn/BkxKfTzqw
+
+ */
+
+'use strict';
+
+require('dotenv').config();
+const dynamodb = require('../../../../utils/dynamoDb');
+const Web3 = require("web3");
+const moment = require("moment");
+const delay = require("delay");
+const _ = require("lodash");
+const vaults = require("./vaults");
+const EthDater = require("./ethereum-block-by-date.js");
+const { delayTime } = require("./config");
+
+const { getHoldings } = require('./getHoldings');
+
+const db = dynamodb.doc;
+const archiveNodeUrl = process.env.ARCHIVENODE_ENDPOINT;
+const infuraUrl = process.env.WEB3_ENDPOINT;
+const archiveNodeWeb3 = new Web3(archiveNodeUrl);
+const infuraWeb3 = new Web3(infuraUrl);
+const blocks = new EthDater(archiveNodeWeb3, delayTime);
+const axios = require('axios');
+
+
+
+const saveVault = async (data) => {
+  const params = {
+    TableName: "holdings",
+    Item: data,
+  };
+  await db
+    .put(params)
+    .promise()
+    .catch((err) => console.log("err", err));
+  console.log(`Saved ${data.name}`);
+};
+
+
+
+
```

Comment:
Personal opinion but I think 4 blank lines here is a lot, better to have 1 or 2.  Same above as well.

Reply:
removed